pkg/conversion/generator.go

```diff
@@ -700,6 +701,37 @@ func (g *generator) existsConvertionFunction(inType, outType reflect.Type) bool
 	return false
 }
 
+func (g *generator) existsDedicatedConversionFunction(inType, outType reflect.Type) bool {
+	if inType == outType {
+		// Assume that conversion are not defined for "deep copies".
+		return false
+	}
+
+	if g.existsConversionFunction(inType, outType) {
+		return true
+	}
+
+	// TODO(wojtek-t): We should somehow change the conversion methods registered under:
+	// pkg/runtime/scheme.go to implement the naming convention for conversion functions
+	// and get rid of this hack.
+	type typePair struct {
+		inType  string
+		outType string
+	}
+	defaultConversions := []typePair{
```

Comment:
You may want to pull this out into a private var in package, I don't know whether the Go compiler automatically inlines this, but it's more of a package level thing than specific to this function.